packages/cli/src/commands/doctor/healthchecks/__tests__/python.test.ts

```diff
@@ -0,0 +1,69 @@
+import execa from 'execa';
+import {cleanup} from '../../../../../../../jest/helpers';
+import python from '../python';
+import getEnvironmentInfo from '../../../../tools/envinfo';
+import {EnvironmentInfo} from '../../types';
+import {NoopLoader} from '../../../../tools/loader';
+import * as common from '../common';
+
+jest.mock('execa', () => jest.fn());
+
+const mockFetchToTemp = jest.fn();
+jest.mock('@react-native-community/cli-tools', () => {
+  return {
+    fetchToTemp: mockFetchToTemp,
+  };
+});
+
+const logSpy = jest.spyOn(common, 'logManualInstallation');
+
+describe('python', () => {
+  afterAll(() => cleanup('android/build.gradle'));
+
+  let environmentInfo: EnvironmentInfo;
+
+  beforeAll(async () => {
+    environmentInfo = await getEnvironmentInfo();
+  }, 15000);
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('returns a message if Python is not installed', async () => {
+    environmentInfo.Languages.Python = 'Not Found';
+    ((execa as unknown) as jest.Mock).mockResolvedValue({stdout: ''});
+    const diagnostics = await python.getDiagnostics(environmentInfo);
+    expect(diagnostics.needsToBeFixed).toBe(true);
+  });
+
+  it('returns false if Python version is in range', async () => {
+    // @ts-ignore
+    environmentInfo.Languages.Python = {
+      version: '2.7.17',
+    };
+
+    const diagnostics = await python.getDiagnostics(environmentInfo);
+    expect(diagnostics.needsToBeFixed).toBe(false);
+  });
+
+  // envinfo has a special field for reporting Python 3 so no need to check for python 3
```

Comment:
Should this be removed?

Reply:
I left it for future reference in case someone wondered why we weren't testing out of range. I'm OK either way 😊

Reply:
I see!